src/python/platforms/android/emulator.py

```diff
@@ -71,7 +71,7 @@ def run(self):
     environment.set_value('ANDROID_SERIAL', device_serial)
 
     logs.log('Waiting on device')
-    adb.wait_until_fully_booted()
```

Comment:
Rather than lose the drive check, can we instead modify wait_until_fully_booted to customize the checks that are done?

e.g. `def wait_until_fully_booted(check_drive=True, check_package_manager=True, etc...)`

And just set the two that don't work to false here.